src/Dependencies/CodeAnalysis.Metadata/CustomDebugInfoEncoder.cs

```diff
@@ -194,7 +194,10 @@ public void AddTupleElementNames(IReadOnlyCollection<(string LocalName, int Slot
                         builder.WriteInt32(info.SlotIndex);
                         builder.WriteInt32(info.ScopeStart);
                         builder.WriteInt32(info.ScopeEnd);
-                        builder.WriteUTF8(info.LocalName);
+                        if (info.LocalName != null)
```

Comment:
We need to check anbd add a test that the EE handles this well. It shouldn't need the name of the local variable since it has the slot index. Not sure why we include the name of the variable in the first place, perhaps we can always emit empty?